src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html

```diff
@@ -97,6 +97,50 @@
               </p>
             </div>
 
+            <div ng-if="record._source._extra_data.record_matches.length > 1 
+            && record._source._workflow.data_type === 'hep' " ng-init="showPossibleMatches = true">
+                <div ng-show="showPossibleMatches">
+                    <div ng-if="record._source._extra_data.record_matches.length <= 5">
+                        <p class="small-text">Showing possible matches</p>
+                    </div>
+                    <div ng-if="record._source._extra_data.record_matches.length > 5">
+                        <p class="small-text">Showing 5 out of {{ (record._source._extra_data.record_matches.length-1) }} possible matches</p>
```

Comment:
IMHO we should remove the ``-1`` in ``record._source._extra_data.record_matches.length``. Shall we also remove  the hardcoded ``5``? ☺️